.ci/jobs/apm-aws-lambda-smoketests-mbp.yml

```
@@ -0,0 +1,41 @@
+---
+- job:
+    name: library/apm-aws-lambda-smoketests-mbp
+    display-name: AWS Lambda extension Smoke Tests
+    description: AWS Lambda extension Smoke Tests
+    project-type: multibranch
+    concurrent: true
+    script-path: .ci/smoke-tests.groovy
+    scm:
+    - github:
+        branch-discovery: no-pr
+        discover-pr-forks-strategy: merge-current
+        discover-pr-forks-trust: permission
+        discover-pr-origin: merge-current
+        discover-tags: false
+        head-filter-regex: '(main|PR-.*|v\d+\.\d+\.\d+.*)'
```

If `discover-tags: false`, then `|v\d+\.\d+\.\d+.*` won't be needed, as far as I know that's the regex for the tags

The below suggestion, will discover all the PRs and changes in `main`
```suggestion
        head-filter-regex: '(main|PR-.*)'
```